logagg/formatters.py

```diff
@@ -55,7 +61,7 @@ def haproxy(line):
 
     log = {}
     log['client_server'] = _line[5].split(':')[0].strip()
-    log['client_port'] = _line[5].split(':')[1].strip()
+    log['client_port'] = int(_line[5].split(':')[1].strip())
```

Comment:
Please rewrite this code to not depend on using integer indices.

Reply:
Sure @prashanthellina